distributed/scheduler.py

```diff
@@ -182,7 +182,11 @@ def nogil(func):
     EventExtension,
 ]
 
-ALL_TASK_STATES = {"released", "waiting", "no-worker", "processing", "erred", "memory"}
+ALL_TASK_STATES = declare(
+    set,
+    {"released", "waiting", "no-worker", "processing", "erred", "memory"},
+)
+sys.modules[__name__].ALL_TASK_STATES = ALL_TASK_STATES
```

Comment:
Was unable to `import` outside of this module without this last line. Hoping we can get rid of it. Asked here ( https://github.com/cython/cython/issues/3944#issuecomment-767338699 )